shapes/contacts/example2/group/group3-soccer-team.ttl

```diff
@@ -0,0 +1,20 @@
+prefix :       <#>
+prefix vcard:  <http://www.w3.org/2006/vcard/ns#>
+
+:this
+    a vcard:Group ;
+    vcard:fn "Group 1" ;
+    vcard:hasMember <../Person/2d73ccc1-9d76-4398-8840-3c23a8ddf04c/index.ttl#this> ;
```

Comment:
Maybe better to use `vcard:hasMember <person1> , <person2> .` instead of `vcard:hasMember <person1> ; vcard:hasMember <person2> .`?